commerce/domain/tax/cart/calculator/classes/TaxCartCalculatorSampleTest.cls

```diff
@@ -0,0 +1,192 @@
+/**
+ * @description A Sample unit test for TaxCartCalculatorSample.
+ */
+@IsTest
+public class TaxCartCalculatorSampleTest {
+
+    private static final String CART_NAME = 'My Cart';
+    private static final String ACCOUNT_NAME = 'My Account';
+    private static final String WEBSTORE_NAME = 'My WebStore';
+    private static final String DELIVERYGROUP_NAME = 'My Delivery Group';
+    private static final String CART_ITEM1_NAME = 'My Cart Item 1';
+    private static final String CART_ITEM2_NAME = 'My Cart Item 2';
+    private static final String CART_ITEM3_NAME = 'My Cart Item 3';
+    private static final String SKU1_NAME = 'My SKU 1';
+    private static final String SKU2_NAME = 'My SKU 2';
+    private static final String SKU3_NAME = 'My SKU 3';
+    private static final Decimal ESTIMATED_PRICE = 350.00;
+    private static final Decimal ACTUAL_PRICE_SKU1 = 100.00;
+    private static final Decimal ACTUAL_PRICE_SKU2 = 200.00;
+    private static final Decimal ACTUAL_PRICE_SKU3 = 300.00;
+
+    @IsTest
+    static void testCalculate_withEmptyDeliveryAddress() {
```

Comment:
Could we follow the Arrange-Act-Assert pattern established on other test classes? That's as simple as break down method code into three visual sections.
see this [example](https://github.com/bsrilok/commerce-extensibility/blob/main/commerce/domain/pricing/cart/calculator/EstimatedActualPricingCalculatorTest.cls#L24)

Reply:
done